src/routes/+page.svelte

```diff
@@ -4,6 +4,7 @@
 	import SocialHost from '$layout/Hero/SocialHost.svelte';
 	import Wave from '$lib/components/Wave.svelte';
 	import Head from '$lib/components/Head.svelte';
+	import FooterHost from '$layout/Footer/FooterHost.svelte';
```

Comment:
I would suggest to group imports from layout with other imports from layout

Reply:
thanks for your advice. I'm implementing it.